FirebasePerformance/ProtoSupport/generate_protos.sh

```diff
@@ -23,7 +23,7 @@
 readonly DIR="$( git rev-parse --show-toplevel )"
 
 # Current release of nanopb being used  to build the CCT protos
-readonly NANOPB_VERSION="0.3.9.5"
```

Comment:
Firebase and Google are locked to 0.3. We would need to migrate the rest of the company if 0.4 is needed.